pkg/translate/translate_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"istio.io/operator/pkg/version"
 )
 
-func TestProtoToValuesV13(t *testing.T) {
+func TestProtoToValuesV14(t *testing.T) {
```

Comment:
We really want a new function for V14 - for now, feel free to have it just call V13.